docs/proxy.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Proxy
-description: Learn how to anonymously access websites when running web scraping or automation jobs. Prevent IP address-based blocking using IP address rotation.
+description: Learn how to anonymously access websites when running web scraping or automation jobs. Improves data throughput and efficiency of bots, and enables access to websites from various geographies.
```

Comment:
Maybe "Improve data throughput and efficiency of bots, and enable access to websites from various geographies" ?

Reply:
Your change looks good to me. Sorry I'm just getting back to you now, but I must have notifications turned off, so only saw your comments when I was in GitHub!